### PR TITLE
Add jobs branding 

### PR DESCRIPTION
**Pull request by markjamesbutler**
This enables Guardian Jobs branding when clientId=jobs.

screen shot 2016-09-14 at 13 34 19
screen shot 2016-09-14 at 13 34 05
screen shot 2016-09-14 at 13 33 51
screen shot 2016-09-14 at 13 52 08

### CODE REVIEW

**Comment by jranks123** [comment]
👍

**Comment by jamespamplin** [comment]
@markjamesbutler is `public/components/jobs/bg-masthead-72px.png` required?

File: public/components/jobs/guardian-jobs-logo.svg [hunk 1]
```diff
@@ -0,0 +1,76 @@
+<?xml version="1.0" encoding="utf-8"?>
```
Comment (jamespamplin) [inline public/components/jobs/guardian-jobs-logo.svg]: This file should be optimised using svgo:

```sh
npm i svgo
./node_modules/.bin/svgo public/components/jobs/guardian-jobs-logo.svg
```

**Comment by jamespamplin** [comment]
:+1: a couple of comments, but otherwise looks good

**Pull request merged**